var/spack/repos/builtin/packages/imlib2/package.py

```diff
@@ -28,3 +30,4 @@ class Imlib2(AutotoolsPackage, SourceforgePackage):
     depends_on('libpng')
     depends_on('libid3tag')
     depends_on('libjpeg-turbo')
+    depends_on('pkg-config', type='build')
```

Comment:
```suggestion
    depends_on('pkgconfig', type='build')
```
this is the virtual package so that both pkg-config and pkgconf are supported.

Reply:
Implemented.